.sisyphus/plans/dp-13-setupwizard-atomicity-audit-reliability.md

```
@@ -0,0 +1,359 @@
+**Change**:
+
+Update `phpunit.xml` `<php>` block to mirror the root phpunit.xml's full env set:
+
+```xml
+<php>
+    <env name="APP_ENV" value="testing"/>
+    <env name="APP_MAINTENANCE_DRIVER" value="file"/>
+    <env name="BCRYPT_ROUNDS" value="4"/>
+    <env name="CACHE_STORE" value="array"/>
+    <env name="CACHE_DRIVER" value="file"/>
+    <env name="DB_CONNECTION" value="mariadb"/>
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)